Lombiq.BaseTheme/ResourceManagementOptionsConfiguration.cs

```diff
@@ -11,6 +12,9 @@ public class ResourceManagementOptionsConfiguration : IConfigureOptions<Resource
     private const string Js = WwwRoot + "js/";
     private const string Vendors = WwwRoot + "vendors/";
 
+    // Make sure to keep this exactly the same version as the "bootstrap" dependency in the package.json file.
+    public static readonly Version NpmBootstrapVersion = new(5, 2, 3);
```

Comment:
@DAud-IcI Would it be feasible to ensure the requirement by parsing the _package.json_ file for the bootstrap version used there, maybe using a static constructor?

Reply:
I'm not sure if it's a good idea to do something so time-consuming as reading and parsing a file in a static filed or static constructor. Sounds like the kind of change Zoli would reject in the review, right @Piedone?

Reply:
File access is a no-no in any ctor indeed. It would be lazily loaded on the first access instead, but that needs locking (since `Version` is not an atomic type), which would get us back to essentially serializing requests. So, let's leave it as it is.

Reply:
Ok, thanks.